includes/embeds/class-amp-tiktok-embed-handler.php

```diff
@@ -34,107 +39,79 @@ public function unregister_embed() {
 	 * @param Document $dom DOM.
 	 */
 	public function sanitize_raw_embeds( Document $dom ) {
-		$nodes = $dom->xpath->query( '//blockquote[ contains( @class, "tiktok-embed" ) ]' );
+		$nodes = $dom->xpath->query( '//blockquote[ @cite and @data-video-id and contains( @class, "tiktok-embed" ) and not( parent::amp-tiktok ) ]' );
```

Comment:
Would you consider using the `Extension::TIKTOK` constant here?

```suggestion
		$nodes = $dom->xpath->query(
			sprintf( '//blockquote[ @cite and @data-video-id and contains( @class, "tiktok-embed" ) and not( parent::%s ) ]', Extension::TIKTOK )
		);
```

Reply:
Sure, it doesn't hurt.